lib/cosmos/gui/qt.rb

```diff
@@ -458,17 +458,13 @@ def add_formatted_text(text, color = nil)
       text << "\n"
     end
     if text =~ /<G>/ or color == Cosmos::GREEN
-      text.gsub!(/<G>/, BLANK)
```

Comment:
I wanted to avoid changing the passed in text. Even though above we call  chomp! but that seemed like a special case.